Data Files/MWSE/mods/CraftingFramework/components/CustomRequirement.lua

```diff
@@ -1,6 +1,7 @@
 local Util = require("CraftingFramework.util.Util")
 local config = require("CraftingFramework.config")
 
+---@class CustomRequirement
 local CustomRequirement = {
     schema = {
         name = "CustomRequirement",
```

Comment:
Completely unrelated to this PR, but it looks like there isn't a `name` field here but it is used in `CustomRequirement:getName()`.